γプレゼンで使う用に調整

kakeru4 opened the pull request:
#105 の作業になります。
今までの作業をγブランチでまとめましたので確認をお願いします。
結構個人的に書いてしまっていますので変更して欲しい場所やここはちがうなどの意見などあればお願いします。
確認したらチェックしてください。
全員がチェックしましたら@Sharoku-hagaさんにマージしてもらいます。

- [x] 芳我
- [x] 川口
- [x] 市川

kakeru4 committed:
[enh#106]Bulletクラス

- 一旦上げ

- 一旦上げ

- 今日の作業

- いらないから削除

- 入れ忘れ

- コメント追加

- 修正しわすれ

kakeru4 committed:
Merge pull request #101 from TeamT-SEN/Data

BattleStateManagerとBattleDataManagerファイル作成

Sharoku-haga committed:
[enh #102]Monsterクラス

-Monsterクラスの一部実装
-MonsterManagerクラス実装
-モデル追加

Sharoku-haga committed:
Merge pull request #104 from TeamT-SEN/Bullet

Bulletクラスの実装

kakeru4 committed:
とりあえず実装完了

kakeru4 commented:
確認おねがいします！

Sharoku-haga commented on Twar/Twar/SubScene/Battle.h:
```diff
@@ -32,7 +40,15 @@ class Battle :public SubScene
+	Light*                  m_pLight;					//!< ライト
+	FieldManager*           m_pFieldManager;
+	CameraController*       m_pCameraController;
+	LoadingThread*			m_pLoadingThread;
```
LoadingThreadはモノステイトパターンやから、メンバから消しておいてください。

kakeru4 replied:
すみません、消し忘れです。
修正します。

Sharoku-haga commented on Twar/Twar/SubScene/Battle.h:
```diff
@@ -32,7 +40,15 @@ class Battle :public SubScene
 	virtual void Draw();
-
+private:
+	Light*                  m_pLight;					//!< ライト
```
ここのLightはどうする？
今後使う可能性があるならこのままメンバとして残しておいてもいいけど、使わないなら消しておくべきかな。

kakeru4 replied:
今のところ使わなさそうなので消しときます。

Sharoku-haga commented on Twar/Twar/SubScene/Battle.h:
```diff
@@ -32,7 +40,15 @@ class Battle :public SubScene
+	FieldManager*           m_pFieldManager;
+	CameraController*       m_pCameraController;
+	LoadingThread*			m_pLoadingThread;
+	BulletManager*          m_pBulletManager;
```
ここのBulletManagerはBattleで使用されていないので、前方宣言も含めて消しておいてください。

kakeru4 replied:
すみません、消し忘れです。
修正します。

kakeru4 committed:
[enh #107]作業完了

- リリースの設定完了

- 修正完了

- デバック版のパス修正

- 弾のテクスチャー追加

- 衝突いている一部修正

- 衝突している部分の修正完了

- 衝突部分の修正

- 修正完了

- 修正完了

Sharoku-haga commented:
修正確認致しましたので、チェックボックスのチェックいれておきました。

kakeru4 commented:
一旦マージします。

kakeru4 commented:
衝突している部分を全部修正しましたのでマージお願いします。
リベースも完了しています。

Sharoku-haga commented:
マージしました

kakeru4 commented:
マージしましたのでcloseします。
